contracts/english_auction_rev1.1.scilla

```diff
@@ -947,6 +995,26 @@ transition End(
         (* payment tokens for the seller *)
         AddToPaymentTokenBalance seller payment_token_address seller_profit;
 
+        (* Paying out a commission if there is one *)
+        match is_commission_fee with
+          | True =>
+            AddToPaymentTokenBalance brand_recipient payment_token_address brand_commission_fee;
+
+            msg_to_collection_contract = {
+              _tag: "TokenSaleCallback";
+              _recipient: collection_contract;  (* TODO: get this address from somewhere *)
```

Comment:
@sam-zilliqa I believe the comment is no longer valid

Reply:
Agree, updated